.changeset/contextPath.md

```diff
@@ -0,0 +1,9 @@
+---
+"@sap-ux/fe-fpm-writer": patch
+"@sap-ux/types": patch
+---
+
+feat: contextPath for new pages
+
+Feature enhacement: in addition to the entity (set) you now can define a contextPath that gets written instead of entitySt then into the `options > settings` of the new page.
```

Comment:
```suggestion
Feature enhancement: in addition to the entity (set) you now can define a contextPath that gets written instead of entitySet then into the `options > settings` of the new page.
```

Reply:
done